utils/auth.js

```diff
@@ -0,0 +1,10 @@
+import jwt from 'jsonwebtoken';
+
+export function verifyToken(req) {
+  const authHeader = req.headers.get('authorization');
+  const token = authHeader?.split(' ')[1];
```

Comment:
Before taking the second element after the split, we should verify that the array is not empty. 
Also, where the space will come from? What is the expected value of the header?